api/models.go

```diff
@@ -147,6 +147,7 @@ type CreateLightningAddressRequest struct {
 type InitiateSwapRequest struct {
 	SwapAmount  uint64 `json:"swapAmount"`
 	Destination string `json:"destination"`
+	IsSending   bool   `json:"isSending"`
```

Comment:
This is weird naming in my opinion: `IsSending`.

Also `InitiateSwapRequest` should be `InitiateSwapOutRequest` right?

Reply:
Should `IsSending` be `UseExactReceiveAmount` or something like that?

Reply:
Changed to `UseExactReceiveAmount `